pkgs/servers/matrix-registration/default.nix

```diff
@@ -0,0 +1,71 @@
+{ stdenv, pkgs, fetchFromGitHub, python3Packages }:
+
+with python3Packages;
+let
+  # officially supported database drivers
+  dbDrivers = [
+    psycopg2
+    # sqlite driver is already shipped with python by default
+  ];
```

Comment:
```suggestion
let
  # officially supported database drivers
  dbDrivers = with python3Packages; [
    psycopg2
    # sqlite driver is already shipped with python by default
  ];
```

Reply:
oh shouldn't have applied this, since buildPythonPackage is also part of python3packages